common/setup.py

```diff
@@ -4,7 +4,7 @@
 
 from setuptools import find_packages, setup
 
-__version__ = '0.15.2'
+__version__ = '0.15.3'
```

Comment:
Do you have to update `requirements-common.txt`?

Reply:
done

Reply:
actually not cause that has to happen after release is published